### DISCUSSION TITLE
prisma is unable to run int hsi browser environemt calling server component inside cleint component

### DISCUSSION THREAD

**Discussion by wailbentafat** [prisma/prisma#25918] [Q&A]
### Bug description

prisma is unable to run int hsi browser environemt calling server component inside cleint component 

### Prisma information

```prisma
// Add your schema.prisma
```

```ts
// Add your code using Prisma Client
```

### Environment & setup

- OS: 
- Database: 
- Node.js version: 

### Prisma Version

```

```

**Comment by ludralph** [comment]
Hi @wailbentafat 

The error message `PrismaClient is unable to run in this browser environment` occurs when you are attempting to use Prisma in a client-side context. This can happen if you're importing and using PrismaClient in a client component or if server-side code containing Prisma operations is being bundled for the browser.
To resolve this issue:

- Ensure that all Prisma operations are performed in server-side code or API routes.
- If you're using Next.js, make sure to use Prisma only in server components or API routes, not in client components.
- If you need to fetch data for a client component, create an API endpoint that uses Prisma on the server-side, then call that endpoint from your client component.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
